packages/docs/docs/getting-started/what-can-you-build-with-unirep.md

```diff
@@ -0,0 +1,29 @@
+---
+
+title: what can you build with unirep
+---
+
+# 🏗️ What Can You Build With UniRep
+
+UniRep is a generalized protocol that allow developers to build many different kinds of applications while keep user data private. Here are a few examples to illustrate the broad range of applications that could be built:
+
+Peer-to-Peer Anonymous Lending:
+ - A problem in current decentralized protocols is that most of them rely on the credit system being over-collateralized. With UniRep, the reputation value will be the credit-score of a prospective borrower that will change based on if they follow the credit rules of the prospective system.
```

Comment:
The reason decentralized lending is over-collateralized is because the lender's do not have an idea of  who the borrower is let alone what their credit-worthiness is. So, the over-collateralization is like a "fee" a user pays for being pseudoanonymous.

With UniRep, a user can receive some defined data attributes that can prove that they are a credit-worthy borrower therefore null-ing the over-collateralization requirement... this is a broad brush of the difference because creating your own credit system is rather complex.

Read more about the problem [here](https://www.newsbtc.com/press-releases/the-danger-of-over-collateralized-loans-to-defi-and-solo-protocols-new-type-of-loan/).